nbproject/project.properties

```diff
@@ -0,0 +1,7 @@
+include.path=${php.global.include.path}
```

Comment:
This file should not be included in the project

Reply:
Solved